helm/examples/azure/sa/README.md

```diff
@@ -0,0 +1,58 @@
+# HPCC Azure storage with a Storage Account
+
+## Directory contents
+
+### Introduction
+Persistent storage can be created manually or through Kubernetes Clusters. If created through Kubernetes the storage will be gone when the Persistent Volume (PV) or Kubernetes clusters are deleted. For Azure a manually created shared storage with a storage account will not depend on any Kubernetes cluster. The data generated will be persistent and re-used as long the PV/PVC names and mount points are the same.
+
+### Create a share with a storage account
+To create a shared storage including storage account you can reference Reference: https://docs.microsoft.com/en-us/azure/aks/azure-files-volume.
```

Comment:
remove one instance of the word "reference Reference:"

Reply:
fixed